src/jit_no_ir.h

```diff
@@ -217,18 +201,20 @@ class EmitX64 {
             }
         }
 
-               // Check for interrupts.
+        // Check for interrupts.
         for (std::size_t i = 0; i < 3; ++i) {
-            if (interrupt_pending[i].exchange(false)) {
```

Comment:
Was the atomic qualifier for this ever actually necessary 🤔

Reply:
I don't think teakra even uses any threads. My theory is that atomics were used for the convenient exchange methods, but given this is a hot path, we'd rather avoid the atomic operation

Reply:
Actually thinking about it, this may be used in the case of multithreaded teakra, where pipe reads happen on the emuthread that may cause an interrupt while teakra is running on a separate thread. Might need to add this back